Sources/SwiftRefactor/PackageManifest/AddPluginUsage.swift

```diff
@@ -0,0 +1,62 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftParser
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+/// Add a plugin usage to a particular target in the manifest's source
+/// code.
+public struct AddPluginUsage: ManifestEditRefactoringProvider {
+  public struct Context {
+    public let targetName: String
+    public let pluginUsage: PackageTarget.PluginUsage
+
+    public init(targetName: String, pluginUsage: PackageTarget.PluginUsage) {
+      self.targetName = targetName
+      self.pluginUsage = pluginUsage
+    }
+  }
+
+  /// The set of argument labels that can occur after the "plugins"
+  /// argument in the Target initializers. (There aren't any right now)
+  private static let argumentLabelsAfterPluginUsages: Set<String> = []
```

Comment:
Do we need to keep this 🤔?

Reply:
I don't think so, let's remove it for now.